.circleci/config.yml

```diff
@@ -30,6 +34,7 @@ commands:
         type: string
     steps:
       - setup-build-workspace
+      - set-deployment-environment
```

Comment:
Turns out we need to set deployment environment as this is where we get the build params now that determine whether we include source code or not in builds.